app/modules/sightings/resources.py

```diff
@@ -531,3 +533,52 @@ def delete(self, sighting):
         # TODO handle failure of feather deletion (when edm successful!)  out-of-sync == bad
         sighting.delete_cascade()
         return None
+
+
+@api.route('/<uuid:sighting_guid>/featured_asset_guid')
+@api.resolve_object_by_model(Sighting, 'sighting')
+class FeaturedAssetGuidBySightingID(Resource):
+    """
+    Featured Asset guid set and retrieval.
+    """
+
+    @api.login_required(oauth_scopes=['sightings:read'])
+    @api.permission_required(
+        permissions.ObjectAccessPermission,
+        kwargs_on_request=lambda kwargs: {
+            'obj': kwargs['sighting'],
+            'action': AccessOperation.READ,
+        },
+    )
+    def get(self, sighting):
+        """
+        Get featured Asset guid.
+        """
+        return {'featured_asset_guid': '%s' % sighting.get_featured_asset_guid()}
```

Comment:
Shouldn't this be done using a schema? 
Likewise the post.
We seem to be hand generating the responses a lot instead of using schemas which I thought was the expected flask way to do this.